test/action.spec.js

```diff
@@ -0,0 +1,112 @@
+const chai = require('chai');
+const expect = chai.expect;
+const sinon = require('sinon');
+const mock = require('mock-require');
+const log = require('npmlog');
+const path = require('path');
+
+const action = (config, args) => require('../src/action')(config, args || []);
+
+log.level = 'silent';
+
+describe('action', () => {
+
+  beforeEach(() => {
+    delete require.cache[require.resolve('../src/action')];
```

Comment:
Quick example:
```js
mock('../src/ios/registerNativeModule', noop)
require('./src/actions.js)() // this gets `noop` mocked, OK

mock('../src/ios/registerNativeModule', noop2)
require('./src/actions.js)() // this gets `noop` instead of noop2, BAD
```